src/uiTest/resources/project.rhino.js

```diff
@@ -402,6 +402,18 @@ function Project(underlyingProject) {
     invokeActionAndWait('GitMachete.CheckoutSelectedAction', ACTION_PLACE_CONTEXT_MENU, { SELECTED_BRANCH_NAME: branchName });
   };
 
+  this.checkoutNextBranch = function() {
+    invokeActionAndWait('GitMachete.NextBranchCheckoutAction', ACTION_PLACE_CONTEXT_MENU,{});
```

Comment:
To be checked if e.g. an empty string can be used as `actionPlace` (as `null` is forbidden, the underlying IntelliJ API method has a `@NonNull` param)